frontend/ui/src/actions/conversations/index.ts

```diff
@@ -87,8 +88,8 @@ export const getConversationInfo = (conversationId: string, retries?: number) =>
       dispatch(mergeConversationsAction([response]));
       return Promise.resolve(true);
     })
-    .catch(async (error: Error) => {
-      if (retries > 5) {
+    .catch(async (error: ConversationInfoError) => {
+      if (retries > 5 || error.status === 404) {
         return Promise.reject(error);
       } else {
```

Comment:
if the conversation Id is not found (invalid match.params), we return and do not fetch again to avoid loops

Reply:
The loop is necessary to find the conversation when we get a websocket notification. If we remove this loop we won't be able to display conversations that come from ws because sometimes some metadata updates arrive even before the actual conversation.